sklearn/svm/_classes.py

```diff
@@ -538,7 +538,8 @@ class SVC(BaseSVC):
         inversely proportional to C. Must be strictly positive. The penalty
         is a squared l2 penalty.
 
-    kernel : {'linear', 'poly', 'rbf', 'sigmoid', 'precomputed'}, default='rbf'
+    kernel : {'linear', 'poly', 'rbf', 'sigmoid', 'precomputed'} or callable,  \
+        default='rbf'
```

Comment:
With this update do you think we need the follow anymore?

https://github.com/scikit-learn/scikit-learn/blob/bacc91cf1d4531bcc91aa60893fdf7df319485ec/sklearn/svm/_classes.py#L543-L544

I think it is safe to remove.

Reply:
Good catch, removed in latest commit!